core/iwasm/aot/aot_loader.c

```diff
@@ -674,6 +678,53 @@ load_custom_section(const uint8 *buf, const uint8 *buf_end, AOTModule *module,
                                    error_buf, error_buf_size))
                 goto fail;
             break;
+#if WASM_ENABLE_LOAD_CUSTOM_SECTION != 0
+        case AOT_CUSTOM_SECTION_RAW:
+        {
+            const uint8 *p_orig = p;
+            char section_name_buf[32];
+            const char *section_name;
+            uint32 name_len, buffer_len;
+            WASMCustomSection *section;
+
+            if (p >= p_end) {
+                set_error_buf(error_buf, error_buf_size, "unexpected end");
+                goto fail;
+            }
+
+            read_string(p, p_end, section_name);
+
+            name_len = p - p_orig;
```

Comment:
Should be `strlen(section_name)`? `p - p_orig` includes 2 bytes string length. section_name is already '\0' terminated.